packages/devcmd/src/from-cli/index.ts

```diff
@@ -104,31 +104,56 @@ const scriptRunners = [
 ];
 
 async function findAndRunScript(devCmdsDir: string, commandName: string, commandArgs: string[]): Promise<void> {
```

Comment:
IMO the function body of ```findAndRunScript``` gets harder and harder to read with each PR 😄 maybe we should consider adding some comments or extracting the logic into functions with descriptive names

Reply:
that's one reason why I split out `tryRunScript` as a function :D

I agree that it has still grown a bit, but I don't think it's too complicated for now. So I'll kick this can down the road for a little while longer and leave it as it is for now :)